faqs/how-do-i-get-involved.md

```diff
@@ -0,0 +1,10 @@
+---
+title: How can I get involved
+weight: 9.1
+attribution:
+  name: sum guy on reddit
+  link: /u/twoofspades
```

Comment:
Please provide a full url (https://...) to your profile

Reply:
Any way to stay mostly anon?